.vscode/launch.json

```diff
@@ -19,9 +19,25 @@
                 "PROXY_ENABLE_BASIC_AUTH": "true",
                 // set insecure options because we don't have valid certificates in dev environments
                 "OCIS_INSECURE": "true",
+                // set some hardcoded secrets
```

Comment:
I'm not the vscode guy, so I don't really mind :stuck_out_tongue:  But does that work? Afaik it isn't valid json if it has comments...

Reply:
Yes, vscode json can handle comments

Reply:
it is called JSON5 https://json5.org/